docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
+
+.. figure::  images/histogram.png
+   :align:   center
+
+An appropriate cutoff would exclude clear outliers, but retain most of the samples. Here we have already removed blanks from our data and eliminated the outliers prior to analysis so we will just use the minimum number of features observed in our samples (11030) as the cutoff.
+
+To rarefy the data, select "Rarefy" from the dropdown menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/rarify_parameter_without_number.png
+   :align:   center
+
+Several parameters will have only one option which will be automatically selected for you. In the "p-sampling-depth" field we will specify the number of features to rarefy our samples to. Enter "11030" in this box, and click "Add Command". 
+
+.. figure::  images/rarify_parameter_with_sampling_depth.png
+   :align:   center
 
-You can get a good idea of where to set this threshold by looking at the
-histogram generated by summarizing the input closed-reference OTU table, as
-discussed in :doc:`qiita-16S-analysis`. Here, it looks like 2100 would be an
-appropriate cutoff: it excludes one clear outlier, but retains most of the
-samples.
+Click the run button to start the process of rarefaction. The view will return to the original screen, while the rarefaction job runs. Refresh your browser every 10-20 seconds until the rarefied biom table appears: 
 
-.. figure::  images/analysis-closed-ref-histogram.png
+.. figure::  images/rarify_workflow.png
    :align:   center
 
-Enter 2100 in the rarefaction depth field, select the check boxes for
-Summarize Taxa and Beta Diversity, and click "Start
-Processing". You will see a list each step in the analysis, followed by its
-status:
+Select the newly generated "dflt_name - BIOM" artifact. Typically, we could click generate summary again to view a histogram of the rarefied samples but in this case the samples will not have been changed, but it is still the best practice to always rarefy your data. Once the data is rarefied, we can begin analyzing the data.
+
+Taxa Bar Plots
+~~~~~~~~~~~~~~
+
+When creating a closed reference BIOM table, each sequence is matched to the Green Genes database with 97% accuracy, and assigned a taxonomy (See this section for a refresher on '16S data <http://cmi-workshop.readthedocs.io/en/latest/qiita-16S-processing.html>`__). This enables us to display this data to view the percentage of each taxa within each sample. When using "Deblurred" data, there is no taxa assignment since features are kept as individual error-corrected sequences, so if you are referencing this tutorial with your own deblurred data you can skip to the next section `Alpha Diversity Analysis`_.
```

Comment:
Alpha diversity analysis link is also displayed incorrectly... and does not work, at least in review version.